src/pages/api/venues/[venueId].ts

```diff
@@ -0,0 +1,27 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import axios from "axios";
+
+import formatResponseObject from "../../../util/api/formatResponseObject";
+import getSourceUrl from "../../../util/api/getSourceUrl";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const { venueId } = req.query;
+  const [source, id] = (venueId as string).split(":");
+
+  if (!source || !id) {
+    res.status(400).json({ msg: "Invalid ID parameter" });
```

Comment:
Wondering why we are using `msg` instead for instance `message`?

Reply:
This is just a habit of mine that I have used on my own projects. I'll change these to `message`.